monai/networks/nets/hovernet.py

```diff
@@ -52,6 +62,7 @@ def __init__(
         act: Union[str, tuple] = ("relu", {"inplace": True}),
         norm: Union[str, tuple] = "batch",
         kernel_size: int = 3,
+        padding: int = 0,
```

Comment:
Is it different with the docstring?

Reply:
Thanks @KumoLiu , I made a mistake here.